CensusAnalytics/OpenDataLoader/ODL.Common/IngestBase.cs

```diff
@@ -43,7 +43,7 @@ public IngestBase(LogHandler LogObject, ODL.Common.DBConnectionDetails DbConnect
         /// <returns>Number of new records loaded from selected file.</returns>
         public Int32 StartLoading()
         {
-            CreateDatabaseTable();
+            //CreateDatabaseTable();

```

Comment:
I know I did this. I really don't know what I'm going to do with this method; it seems unnecessary.

Reply:
It's probably unnecessary if you already have a table with stuff in it, so just noop if it already exists... otherwise use that method to create the table structure inside your DB?